scripts/eurostat/health_determinants/alcohol_consumption/process.py

```diff
@@ -119,26 +147,41 @@ def _rename_frequency_column(self, df: pd.DataFrame) -> pd.DataFrame:
         return df.rename(columns={'frequenc': 'frequenc_alcohol'})
 
 
-if __name__ == '__main__':
-    input_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
-                              "input_files")
-    ip_files = os.listdir(input_path)
-    ip_files = [input_path + os.sep + file for file in ip_files]
+def main(_):
+    mode = _FLAGS.mode
+    global import_name
+    import_name = "alcohol_consumption"
+    if mode == "" or mode == "download":
+        EuroStatAlcoholConsumption.download_data(import_name)
+    if mode == "" or mode == "process":
+        try:
+            input_path = os.path.join(
+                os.path.dirname(os.path.abspath(__file__)), "input_files")
+            ip_files = os.listdir(input_path)
+            ip_files = [input_path + os.sep + file for file in ip_files]
+
+            # Defining Output Files
+            data_file_path = os.path.join(
+                os.path.dirname(os.path.abspath(__file__)), "output")
+
+            csv_name = "eurostat_population_alcoholconsumption.csv"
+            mcf_name = "eurostat_population_alcoholconsumption.mcf"
+            tmcf_name = "eurostat_population_alcoholconsumption.tmcf"
 
-    # Defining Output Files
-    data_file_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
-                                  "output")
+            cleaned_csv_path = os.path.join(data_file_path, csv_name)
+            mcf_path = os.path.join(data_file_path, mcf_name)
+            tmcf_path = os.path.join(data_file_path, tmcf_name)
 
-    csv_name = "eurostat_population_alcoholconsumption.csv"
-    mcf_name = "eurostat_population_alcoholconsumption.mcf"
-    tmcf_name = "eurostat_population_alcoholconsumption.tmcf"
+            loader = EuroStatAlcoholConsumption(ip_files, cleaned_csv_path,
+                                                mcf_path, tmcf_path,
+                                                import_name)
+            loader.generate_csv()
```

Comment:
Improved sentence:

Please add try-catch blocks to the `generate_csv`, `generate_mcf`, and `generate_tmcf` methods within the `EuroStat` class in the common folder.

Reply:
Addressed